footer.php

```diff
@@ -1,6 +1,6 @@
 <footer class="site-footer">
             <div class="row">
-                <p>Copyright 2020 | Developer links: 
+                <p>Copyright 2022 | Developer links: 
```

Comment:
There is a way to make the copyright year dynamic so you don't have to change it every year.